core/autoscaling.go

```diff
@@ -6,6 +6,7 @@ import (
 	"strings"
 	"time"
 
+	"errors"
```

Comment:
I just realised, it might be because you used goimports while I was using go fmt in my hook. Which is why the formatting were fighting